cdap-ui/app/cdap/components/AdminMetadataPane/index.js

```diff
@@ -30,13 +31,32 @@ function AdminMetadataPane({ statObject }){
   let statsList = [];
 
   statObject.stats.forEach((stat) => {
-    statsList.push (
-      <StatContainer
-        label={T.translate(`features.Management.DetailPanel.labels.${stat.statName}`)}
-        number={stat.statNum}
-        key={shortid.generate()}
-      />
-    );
+
+    //Ignore Stats Associated with WritePointer, ReadPointer, and VisibilityUpperBound
+    if(!(stat.statName === 'SnapshotTime')) {
+
+      //Convert snapshot time to human readable number
+      if(stat.statName === 'WritePointer' || stat.statName === 'ReadPointer' || stat.statName === 'VisibilityUpperBound'){
```

Comment:
ok I got it. never mind. got it. this could still be simplified a little bit though